src/xrServerEntities/script_storage.cpp

```diff
@@ -325,6 +347,53 @@ void disable_os_funcs(lua_State* L)
 	lua_pop(L, 1);
 }
 
+bool LoadKernelScriptToGlobal(lua_State* L, const char* name)
+{
+	string_path FileName;
+	xr_string FixedFileName = name;
+	//FixedFileName = "kernel\\" + FixedFileName; //When this line is enabled all lua files are not being loaded after loading a game
```

Comment:
When I put `lua` files into `scripts/kernel` directory they become unavailable in lua scripts after new/loading a game.
But when I put them just into `scripts` - those files are loaded into game and I can use them.
I.E. with this line I can run in console `run_string debugger_attach()` where `debugger_attach` is a function from `global.lua` but after I create a new game same command give me `function not found` error